smallest_multiple.js

```diff
@@ -6,9 +6,25 @@
  * @return { Number }         Lowest Positive Number that is evenly divisible by all numbers
  *                            between 1 and `ceiling`
  */
-module.exports = function( ceiling ) {
+module.exports = function(ceiling) {
   // do work here
+  let i = ceiling;
 
-
-  return 0;
-};
\ No newline at end of file
+  // if not 0 then i++
+  function checkI(i) {
+    for (let j = 1; j <= ceiling; j++) {
+      // loop ceiling % i   where i goes from 1 to 20  must % = 0
+      if (i % j !== 0) {
+        return false;
+      }
+    }
+    return true;
+  }
+  // break loop when i% numbers through ceiling are 0
+  while (!checkI(i)) {
+    checkI(i);
```

Comment:
Since you're already checking I in the parenthesis of the while loop, you don't need to do it again on this line.